diff --git a/handlers/adminViewHandler.js b/handlers/adminViewHandler.js
index 9e5327b..688ce8f 100644
--- a/handlers/adminViewHandler.js
+++ b/handlers/adminViewHandler.js
@@ -37,6 +37,10 @@ exports.booksByTitle = function(req, res) {
     });
 };
 
+exports.editBook = function(req, res) {
+    res.render('editBook');
+};
+
 exports.addNewBook = function(req, res) {
     res.render('addNewBook');
 };
diff --git a/public/javascripts/main.js b/public/javascripts/main.js
index 9c94fb5..6f93075 100644
--- a/public/javascripts/main.js
+++ b/public/javascripts/main.js
@@ -7,13 +7,11 @@
 // decide if each row would have an 'id' that would serve as an index as well
 // as well as the corresponding buttons.
 function getRowData(caller, tableId) {
-    var dataTbody = document.getElementById(tableId).tbodies;
-    var calleeIndex = (caller.parentNode.parentNode.cells[0].innerHTML);
-    alert(calleeIndex);
-    console.log(calleeIndex);
+    var calleeParentNode = caller.parentNode;
+    var rowData = calleeParentNode.parentNode.cells;
+    return rowData; //  HTMLCollections (array)
 };
 
-// function editBookEntry() {
-//     var bookData = {};
-//     
-// };
+function editBookData() {
+    /* Make ajax call, sending the contents of the row array in the req/res */
+};
diff --git a/routes.js b/routes.js
index 5d98067..41d7233 100644
--- a/routes.js
+++ b/routes.js
@@ -25,4 +25,8 @@ module.exports = function(app) {
     app.get('/employeeList', admin.employeeList);
     app.get('/addNewEmployeeList', admin.addNewEmployee);
     app.get('/logOut', admin.logOut);
+
+    /* For testing purposes only */
+    app.get('/editbook', admin.editBook);
+    // app.post(); // sending data to the server from the page itself
 };
diff --git a/views/editBook.jade b/views/editBook.jade
index 030f82e..92895b6 100644
--- a/views/editBook.jade
+++ b/views/editBook.jade
@@ -6,21 +6,21 @@ block content
     table
         tr
             td ISBN/Book Number:
-            td: input#isbnInput #{bookNumber}
+            td: input#isbnInput(type='text',value='123123123', disabled='true')
         tr
             td Title:
-            td: input#titleInput #{title}
+            td: input#titleInput(type='text',value='Javascript')
         tr
             td Author/s:
-            td: input#authorInput #{author}
+            td: input#authorInput(type='text',value='Me')
         tr
             td Publisher/s:
-            td: input#publisherInput #{publisher}
+            td: input#publisherInput(type='text',value='you')
         tr
             td Date Published:
-            td: input#publishDateInput #{publishDate}
+            td: input#publishDateInput(type='text',value='me')
         tr
             td Quantity:
-            td: input#quantityInput #{quantity}
+            td: input#quantityInput(type='text',value='you', disabled='true')
         tr
             td: button#saveChanges Save
